python/cudf/cudf/tests/test_indexing.py

```diff
@@ -1343,3 +1343,21 @@ def test_loc_zero_dim_array():
 
     assert_eq(psr[np.array(0)], gsr[np.array(0)])
     assert_eq(psr[np.array([0])[0]], gsr[np.array([0])[0]])
+
+
+@pytest.mark.parametrize(
+    "arg",
+    [
+        slice(None),
+        slice((1, 2), None),
+        slice(None, (1, 2)),
+        (1, 1),
```

Comment:
Should we add negative test for tuple size > lvl in multi-indexing

Reply:
Added better validation of the tuple and some tests